bayesian_asimetrix/statistics.py

```diff
@@ -48,11 +48,8 @@ def max_post_estimate(df_post: pd.DataFrame) -> pd.Series:
         MAP
     """
 
-    dp_post = df_post.copy()
-    params = dp_post.columns
-    kde = stats.gaussian_kde(dp_post.values.T)
-    dp_post['pdf'] = kde.pdf(dp_post.values.T)
-    map_theta = dp_post[dp_post.pdf == dp_post.pdf.max()].iloc[0][params]
+    pdfs = prob_density_estimate(df_post)
+    map_theta = df_post.iloc[np.where(pdfs == pdfs.max())]
     map_theta.name = 'MAP'
```

Comment:
Si map_theta es un dataframe es mejor definirlo como 
```suggestion
    map_theta['name']= 'MAP'
```
Tengo entendido que en algunas versiones de python/pandas esa notación puede fallar